README.md

```diff
@@ -2,10 +2,10 @@
 
 ## Capstone Project of the IT bootcamp at neuefische
 
-CatYummy allows you to track your cat's decisions about its food. After adding the food, your cat likes or not, CatYummy will help your memory out when you will stand in front of a long row with cat food products next time, trying to remember which food your cat has already tested...and didn' t like it. It will save your money, helps to reduce waste and makes your cat happier.
+CatYummy allows you to track your cat's decisions about its food. After adding the food, your cat likes or not, CatYummy will help your memory out next time when you will stand in front of a long row with cat food products in the supermarket or pet, trying to remember which food your cat has already tested...and didn' t like it. CatYummy will save your money, helps to reduce waste and makes your cat happier.
 
```

Comment:
"CatYummy will save your money, helps to reduce waste and makes your cat happier." nice slogan 👍

Reply:
I would add an "which" after "After adding the food, which your cat likes or not..."
and
"...in the supermarket or pet" you might can keep pet away!?

Reply:
many thanks 😄, the "shop" from "pet shop" has moved elsewhere i dont know